modules/base-commerce/composables/useAppVersionCheckAndForceReload.ts

```diff
@@ -0,0 +1,30 @@
+import BUILD_TIME_VERSION from '#ioc/config/BUILD_TIME_VERSION'
+import useCheckAppVersion from '#ioc/services/useCheckAppVersion'
+import useShowErrorNotification from '#ioc/composables/useShowErrorNotification'
+import useI18n from '#ioc/composables/useI18n'
+import { onMounted, onUnmounted } from 'vue'
+
+export default () => {
```

Comment:
Why is this in the base-commerce module? I get it that its depending on "generic" useCheckAppVersion, but I would still put it inside build-time-version module.